woo-razorpay.php

```diff
@@ -568,6 +569,14 @@ private function enqueueCheckoutScripts($data)
             wp_register_script('razorpay_wc_script', plugin_dir_url(__FILE__)  . 'script.js',
                 array('razorpay_checkout'));
 
+            $mod_version = get_plugin_data(plugin_dir_path(__FILE__).'woo-razorpay.php')['Version'];
+
+            $data['_'] = array(
+                    'integration'           => 'woocommerce',
+                    'integration_version'   => $mod_version,
+                    'integration_parent_version' => WOOCOMMERCE_VERSION,
```

Comment:
Can you confirm if this is defined in all of our supported versions?

Reply:
This "WOOCOMMERCE_VERSION" const is present in first release as well, so it's supported by all of our plugins versions.